lib/data_breach.rb

```diff
@@ -53,6 +53,7 @@ def data_breach(report, logged_in_user)
       from = MailHandler.address_from_name_and_email(
         'WhatDoTheyKnow.com data breach report', blackhole_email
       )
+      set_reply_to_headers(nil, 'Reply-To' => report.contact_email)
```

Comment:
The `contact_email` field is optional, so we won't necessarily have an email address in `report.contact_email`.

We might also have a `logged_in_user`, so perhaps want to pass in the `logged_in_user` as the first argument, and then if there's an email address in `contact_email`, pass that in as the reply-to header?

Not sure exactly how the blackhole stuff works inside `set_reply_to_headers`, @gbp may be able to give more detailed guidance 🙂

Reply:
Yes on reflection I think you're right, best to leave the logged in user out of this.